ods_ci/utils/scripts/util.py

```diff
@@ -83,8 +83,30 @@ def execute_command(cmd: str, print_stdout: bool = True) -> str | None:
 
 def oc_login(ocp_api_url, username, password, timeout=600):
```

Comment:
Maybe it will be more reasonable to add one more variable `kubeconfig_path` like:
```suggestion
def oc_login(ocp_api_url, username="", password="", kubeconfig_path="", timeout=600):
```
In this way, it is very simple that when the kubeconfig_path is not there we use the user/password and vice versa

Reply:
I prefer keeping the existing `EXTERNAL_KUBECONFIG` env approach for now; adding a separate `kubeconfig_path` arg to oc_login would require extra conditions in generateTestConfigFile.py so the effort is similar. wyt?

Reply:
I find it's more straightforward to understand how a method works by reading its signature and code directly. It's less intuitive when I have to read separate documentation just to discover that its behavior depends on a special environment variable.